app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -23,5 +23,25 @@ def logout_initiated(user_uuid:, unique_session_id:, success:)
         success: success,
       )
     end
+
+    # Tracks when user confirms registration email
+    # @param [Boolean] success
+    # @param [Hash] errors
+    # @param [String] email
+    # @param [Hash] error_details
```

Comment:
:chef-kiss: for including these! 

Though based on the above, I guess this should now read:

```suggestion
    # @param [String] email
    # @param [String] failure_reason
```

Reply:
Filed: https://cm-jira.usa.gov/browse/LG-7134